src/rocky/index.js

```diff
@@ -0,0 +1,42 @@
+// Copyright [2016] Pebble Technology
+var rocky = require('rocky');
+
+var round = require('./round');
+var rect = require('./rect');
+
+function DrawCommand(platform) {
```

Comment:
The require is a directive to webpack to include a module in the bundled JS file. At bundling time, `rocky.watchInfo.platform` doesn't exist (it's an API accessible at runtime) and I'm not sure how webpack would handle this (maybe ignore it?)